docs/modules/ROOT/pages/documentation/user-guides/monitoring/monitoring-console.adoc

```diff
@@ -283,140 +366,158 @@ This is represented by states:
 * _Green_: values in this range indicate a healthy state
 * _White_: values in this range are in no particular state (neither good or bad, maybe undecidable)
 
-image::monitoring-console/mc_settings_alerts.png[Widget Alerts Settings]
+State changes to amber or red are reported as alerts. This show in the widget they affect.
+The settings allow to filter these related alerts to prevent them from showing.
+This makes sense on pages that have dedicated widgets to show alerts or just if users prefer to see the graph instead.
+
+image::monitoring-console/mc_settings_widget_alerts-h.png[Widget Alerts Settings]
 
 [cols="1,1,4",options="header"]
 |====================
 | Property | Default | Description
-| Show | (all checked) | Filters that allow to control the type of alerts that should be shown. Include _Amber_ and/or _Red_ alerts, _Ongoing_ and/or _Stopped_ alerts, _Acknowledged_ and/or _Unacknowledged_ alerts. If for any of these 3 groups of pairs both are unchecked the list will be empty. Widgets of type _Time Curve_ automatically exclude stopped and acknowledged alerts to avoid cluttering the graph with alerts that aren't relevant any more. Use a widget of type _Alerts Table_ to show all alerts according to the filter at all times.
+| Filter | (none checked) | Do not show (when checked) _Amber_ and/or _Red_ alerts, _Ongoing_ and/or _Stopped_ alerts, _Acknowledged_ and/or _Unacknowledged_ alerts.
 |====================
 
+NOTE: If for any of these 3 groups of pairs both are checked no alerts would be shown. Widgets of type _Time Curve_ automatically exclude stopped and acknowledged alerts to avoid cluttering the graph with alerts that aren't relevant any more. Use a widget of type _Alerts Table_ to show all alerts according to the filter at all times.
```

Comment:
```suggestion
NOTE: If for any of these 3 groups of pairs both are checked no alerts would be shown. Widgets of type _Time Curve_ automatically exclude stopped and acknowledged alerts to avoid cluttering the graph with alerts that aren't relevant anymore. Use a widget of type _Alerts Table_ to show all alerts according to the filter at all times.
```